Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -798,12 +818,12 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
                      vertex.getFlags(),
                      truncateFloatFraction(vertex.getChi2(), mCollisionPositionCov),
                      vertex.getNContributors(),
-                     truncateFloatFraction(tsTimeStamp, mCollisionPosition),
+                     truncateFloatFraction(relInteractionTime, mCollisionPosition),
                      truncateFloatFraction(timeStamp.getTimeStampError() * 1E3, mCollisionPositionCov),
                      collisionTimeMask);
     auto& trackRef = primVer2TRefs[collisionID];
     // passing interaction time in [ps]
-    fillTrackTablesPerCollision(collisionID, tsTimeStamp * 1E3, trackRef, primVerGIs, recoData,
+    fillTrackTablesPerCollision(collisionID, interactionTime * 1E3, trackRef, primVerGIs, recoData,
```

Comment:
Was not the `interactionTime` already converted to ns at https://github.com/AliceO2Group/AliceO2/blob/0cd3f782fe75bf1550b47820df7684f3f9e34ff8/Detectors/AOD/src/AODProducerWorkflowSpec.cxx#L793

Reply:
Hi @shahor02. In `fillTrackTablesPerCollision()` interaction time is in `[ps]` since TOF signal is in `[ps]` too (if I'm not mistaken)

Reply:
right, missed that, then forget this comment. But the one about the FPE protection seems to be valid one